src/systemd/sd-messages.h

```diff
@@ -252,6 +252,9 @@ _SD_BEGIN_DECLARATIONS;
 #define SD_MESSAGE_TIME_BUMP                          SD_ID128_MAKE(7d,b7,3c,8a,f0,d9,4e,eb,82,2a,e0,43,23,fe,6a,b6)
 #define SD_MESSAGE_TIME_BUMP_STR                      SD_ID128_MAKE_STR(7d,b7,3c,8a,f0,d9,4e,eb,82,2a,e0,43,23,fe,6a,b6)
 
+#define SD_MESSAGE_TIME_SET_EPOCH                     SD_ID128_MAKE(d9,3d,93,46,71,97,42,bd,ad,92,75,df,52,66,cb,58)
+#define SD_MESSAGE_TIME_SET_EPOCH_STR                 SD_ID128_MAKE_STR(d9,3d,93,46,71,97,42,bd,ad,92,75,df,52,66,cb,58)
```

Comment:
Hmm, this has no associated catalog entry?

What I meant is to reuse the time bump log id (with some appropriate rewording of cource)

Reply:
OK, updated.

Reply:
I dropped the catalog entry because it's fully useless now.